include/zim/uuid.h

```diff
@@ -26,7 +26,7 @@
 #include <algorithm>
 #include <cstring>
 #include <string>
-
+#include "zim.h"
```

Comment:
Do we really need include `zim.h` as we don't use `LIBZIM_API`  ?
Or a `LIBZIM_API` is missing ?